test/tests/functional/pbs_qstat_formats.py

```diff
@@ -409,6 +409,34 @@ def test_qstat_json_valid_multiple_jobs(self):
         except ValueError:
             self.assertTrue(False)
 
+    def test_qstat_json_valid_multiple_jobs_p(self):
+        """
+        Test json output of qstat -f is in valid format when array jobs are
+        queried and make sure that attributes are displayed with `p` option
+        """
+        j = Job(TEST_USER)
+        j.set_sleep_time(100)
+        jid = self.server.submit(j)
+        jid2 = self.server.submit(j)
+        jid3 = self.server.submit(j)
+        self.server.expect(JOB, {'job_state': "R"}, id=jid3)
```

Comment:
Shouldn't we expect the other jobs also to be in R state ?

Reply:
I tried to take a shortcut, but I can change it.